### PR TITLE
keyboard navigation Model3DViewer on focus

### LINKED ISSUE #236: Enable keyboard navigation on Model3Dviewer only when it is in focus
Possible solution explained here #233. This needs to be fixed if #233 is to be approved.

### PR DESCRIPTION
**Pull request by follan**
Fixes #236

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-238.gearboxjs.preview.cogniteapp.com

File: src/components/Model3DViewer/stories/full.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ function ExampleComponent(props) {
 | `assetId`                  | id of asset to highlight                                            | `number`                                                                            |         |
 | `boundingBox`              | bounding box object, that describes dimension of viewed asset nodes | `THREE.Box3`                                                                        |         |
 | `cache`                    | object for caching 3D viewers instances                             | `{ [name:string]: any }`                                                            |         |
-| `enableKeyboardNavigation` | enable keyboard navigation in viewer                                | `boolean`                                                                           |  false  |
+| `enableKeyboardNavigation` | enable keyboard navigation in viewer. Viewer must be in focus       | `boolean`                                                                           |  false  |
```
Comment (mrtraser) [inline src/components/Model3DViewer/stories/full.md]: Could you set `enableKeyboardNavigation={true}` by default? @anderscognite was asking about that and I guess it is safe to use keyboard navigation from now.

Comment (follan) [reply src/components/Model3DViewer/stories/full.md]: Done

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-238.gearboxjs.preview.cogniteapp.com

**Pull request merged**